### PR TITLE
adapted modules to comply with new API

### PR DESCRIPTION
**Pull request by 1over**

### CODE REVIEW

File: modules/timer.py [hunk 1]
```diff
@@ -62,7 +64,7 @@ def timer(bot, update, args, job_queue, chat_data):
         return
     if 'pidor_active' in chat_data:
         pidor_user = str(chat_data['pidor_user'])
-        update.message.reply_text('%s уже запустил таймер' % all_ids[pidor_user])
+        update.message.reply_text('{} уже запустил таймер'.format(all_ids[pidor_user]))
```
Comment (eluppol) [inline modules/timer.py:67]:
```suggestion
        update.message.reply_text(f'{all_ids[pidor_user]} уже запустил таймер')
```

File: modules/bomb_word.py [hunk 2]
```diff
@@ -93,22 +95,21 @@ def trigger_bombers(bot, job_queue, update, chat_data, bombers):
...
         chat_data['bombs'][bomber]['casualties'][user_id] += 1
 
-    job_queue.run_once(remove_pidor, BOMB_PIDOR_TIMEOUT, context={'id': user_id, 'chat_data': chat_data})
+    context.job_queue.run_once(remove_pidor, BOMB_PIDOR_TIMEOUT, context={'id': user_id, 'chat_data': chat_data})
 
     if len(bombers) == 1:
-        update.message.reply_text(
-            'Ты обосрался! Слово \"%s\" было бомбой! Теперь ты пидор на целый день! Л*ОХ' %
+        msg = 'Ты обосрался! Слово \"{}\" было бомбой! Теперь ты пидор на целый день! Л*ОХ'.format(
             chat_data['bombs'][bomber]['word'])
```
Comment (eluppol) [inline modules/bomb_word.py:102]:
```suggestion
        msg = f'Ты обосрался! Слово \"{chat_data['bombs'][bomber]['word']}\" было бомбой! Теперь ты пидор на целый день! Л*ОХ'
```

File: modules/bomb_word.py [hunk 3]
```diff
@@ -50,40 +51,41 @@ def bomb_info_payload_generator(chat_data):
         display_user_name = get_username_by_id(chat_data, user_id)
         # setting casualties info
         casualties_count = 0
-        casualties_list_str = []
+        casualties_list = []
         for user, count in bombinfo['casualties'].items():
-            casualties_list_str.append('%s (%d)' % (get_username_by_id(chat_data, user), count))
+            casualties_list.append('{} ({})'.format(get_username_by_id(chat_data, user), count))
```
[the remark this answers is not in the document]
Comment (1over) [reply modules/bomb_word.py:56]: yes, it's a good idea, but I didn't want to change it in all places within the scope of the current PR

**Review by kramar42 — APPROVED** [review]
looking good

**Pull request merged**